src/commons/modifiers.ts

```diff
@@ -96,26 +96,28 @@ export type ContainerModifiers =
   BorderRadiusModifiers &
   BackgroundColorModifier;
 
-
 export function extractColorValue(props: Dictionary<any>) {
-  // const props = this.getThemeProps();
-  const allColorsKeys: Array<keyof typeof Colors> = _.keys(Colors);
+  const scheme = Appearance.getColorScheme();
+  const schemeColors = Colors.schemes[scheme!];
```

Comment:
How do you know `scheme` in not `null` or `undefined` (here and below)?

Reply:
I don't but technically if it's null or undefined then schemeColors will be undefined and it'll ignore them. 

Im not sure why `Appearance.getColorScheme()` can return undefined or null, maybe the user can set it in the device preferences? 
In this case, I guess we should ignore the schemes, no?

Reply:
I'm not sure why but this is the definition in react-native:
```
type ColorSchemeName = 'light' | 'dark' | null | undefined;
```
Actually it's more weird because the [original code](https://github.com/facebook/react-native/blob/93377ff5089d981e56094329b6f6893b3124be12/Libraries/Utilities/NativeAppearance.js#L14) does not have them (does not seem to have changed in the last two years)...

BTW, you can now remove the `!` since it has no purpose now.

Reply:
fixed